l10n_ar_wsfe/wsfetools/wsfe_easywsy.py

```diff
@@ -56,11 +56,9 @@ def parse_invoice(self, invoice, number=False, invoices=False):
         if not number:
             number = invoice.split_number()[1]
 
-        date_invoice = datetime.strptime(invoice.date_invoice, '%Y-%m-%d')
+        date_invoice = invoice.date_invoice
         formatted_date_invoice = date_invoice.strftime('%Y%m%d')
-        date_due = invoice.date_due and datetime.strptime(
-            invoice.date_due, '%Y-%m-%d').strftime('%Y%m%d') or \
-            formatted_date_invoice
+        date_due = invoice.date_due and invoice.date_due.strftime('%Y%m%d') or formatted_date_invoice
```

Comment:
same as above, but for `date_due`

Reply:
Same as above, already have the obj

Reply:
yeah, my bad, it's changed in the latest Odoo version. Thank you!